pkg/client/clientset/versioned/doc.go

```diff
@@ -0,0 +1,20 @@
+/*
+Copyright The Kubernetes Authors.
```

Comment:
change to Platform9. See other files for reference.

Reply:
Made the requested changes